gmqtt/aioclient.py

```diff
@@ -0,0 +1,91 @@
+import asyncio
+from .client import Client, Message
+from typing import Optional, Tuple
+
+
+class MqttClientProtocol:
```

Comment:
Think about name of that class - because we have already use "Protocol" postfix for classes which are describing network protocols.  So, more preciated name of that class will be *MqttClientWrapper* or something similar to it.